frontend/its-our-call/package.json

```diff
@@ -7,6 +7,7 @@
     "isomorphic-fetch": "^2.2.1",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
+    "react-icons": "^2.2.7",
```

Comment:
Not sure if we will use react-icons forever but we can for now.

Reply:
Oh yeah, I guess we should use our own icons since we have designers, haha